### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by kenzable**

### CODE REVIEW

File: browser/js/common/factories/resource_factory.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+		return $http.post('/api/resources', data)
+		.then(DataFactory.getData);
+	}
+  ResourceFactory.like = function(id) {
+    return $http.put('/api/resources/' + id + '/like');
+  }
+  ResourceFactory.dislike = function(id) {
+    return $http.put('/api/resources/' + id + '/dislike');
+  }
+	return ResourceFactory;
```
Comment (kenzable) [inline browser/js/common/factories/resource_factory.js:33]: fix indent

File: browser/js/common/directives/resource-card/resource-card.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+app.directive('resourceCard', function () {
+    return {
+        restrict: 'E',
+        templateUrl: 'js/common/directives/resource-card/resource-card.html'
```
Comment (kenzable) [inline browser/js/common/directives/resource-card/resource-card.js:4]: consider giving it an isolate scope and passing in the resource

File: browser/js/common/directives/resource-card/resource-card.js [hunk 3]
```diff
@@ -0,0 +1,6 @@
+app.directive('resourceCard', function () {
```
Comment (kenzable) [inline browser/js/common/directives/resource-card/resource-card.js:1]: make sure to inject factory for liking/disliking functions

File: browser/js/search_results/search_results.js [hunk 4]
```diff
@@ -0,0 +1,13 @@
+app.controller('SearchCtrl', function($scope, $stateParams) {
+  $scope.resources = $stateParams.resources;
+});
+
+app.config(function($stateProvider) {
+  $stateProvider.state('searchResults', {
+    url: '/search_results',
+    templateUrl: 'js/search_results/search_results.html',
+    params: {
+      resources: null
```
Comment (kenzable) [inline browser/js/search_results/search_results.js:10]: Great that this is working, let's investigate why

File: package.json [hunk 5]
```diff
@@ -55,6 +57,7 @@
     "karma-mocha-reporter": "^1.0.2",
     "lodash": "^3.9.3",
     "mocha": "^3.0.2",
+    "ng-tags-input": "^3.1.1",
```
Comment (kenzable) [inline package.json:60]: GET RID OF THIS

File: server/app/routes/index.js [hunk 6]
```diff
@@ -2,12 +2,13 @@
...
 
-router.use('/members', require('./members'));
 router.use('/resources', require('./resources'));
+router.use('/tags', require('./tags'));
+router.use('/users', require('./users'));
 
 // Make sure this is after all of
 // the registered routes!
-router.use(function (req, res, next) {
+router.use(function (req, res, next) { //eslint-disable-line
```
Comment (kenzable) [inline server/app/routes/index.js:11]: take out es-link disable

File: server/app/routes/resources/index.js [hunk 7]
```diff
@@ -5,29 +5,46 @@ module.exports = router;
 const db = require('../../../db');
 const Resource = db.model('resource');
 const Tag = db.model('tag');
+const User = db.model('user');
 
-//route would be /api/resources?type=article
+//route would be /api/resources?type=article, /api/resources?tag=javascript
```
Comment (kenzable) [inline server/app/routes/resources/index.js:10]: remove

File: server/app/routes/resources/index.js [hunk 8]
```diff
@@ -38,6 +55,28 @@ router.get('/:id', function(req, res, next) {
     .catch(next);
 });
 
+router.put('/:id/like', function(req, res, next){
+    Resource.findById(req.params.id)
+    .then(function(resource){
```
Comment (kenzable) [inline server/app/routes/resources/index.js:60]: Implement this for guides too

File: server/app/routes/tags/index.js [hunk 9]
```diff
@@ -1,21 +1,21 @@
 'use strict';
 const router = require('express').Router(); //eslint-disable-line new-cap
 module.exports = router;
-
+const Promise = require('bluebird');
```
Comment (kenzable) [inline server/app/routes/tags/index.js:4]: Bluebird no longer needed here

**Review by noeladd — APPROVED** [review]

**Comment by glebec** [comment]
@noeladd @kenzable it doesn't look like any of Mack's suggestions were actually implemented before merging this PR?

**Comment by noeladd** [comment]
@glebec We were all reviewing the code together and decided as a group to go ahead and merge the PR. The comments were more for us to revisit in the near future.

**Comment by glebec** [comment]
OK! In that case you might want to add those as issues (or cards if in trello, waffle, whatever).

**Pull request merged**